Automattic/wp-calypso
# SDK: Add autoprefixer to default webpack config

This PR adds autoprefixer (by using `postcss-loader`) to our default webpack config. It currently affects us only when building Gutenberg extensions using the SDK, because Calypso uses the CLI `postcss` instead at this time. 

~Note: I'm adding a "[Status] Blocked / Hold", just to recall to remove the example CSS before shipping.~ Removed demo CSS in https://github.com/Automattic/wp-calypso/pull/26724/commits/61afb70485e9f690b6e643159407f797f8be820d.

To test:
* Checkout this branch
* Build the Tiled Gallery extension: `npm run sdk:gutenberg -- --editor-script=client/gutenberg/extensions/tiled-gallery/tiled-gallery.jsx`
* Load `client/gutenberg/extensions/tiled-gallery/build/tiled-gallery-editor.css` and verify vendor prefixes have been applied to the result CSS.
* Build Calypso the regular way (`npm start`) and verify there are no differences in the result CSS (you can refer to `/public/style.css` and `/public/style-debug.css`).
